.github/workflows/link-check.yml

```diff
@@ -1,62 +0,0 @@
-name: Link Check
```

Comment:
Non-blocking question: we think link checking of Markdown content doesn't have value? Removing this file gets rid of link checks for site and Markdown in-repo docs. I am fine with that if that is the implied view of the team here based on this.

Reply:
I added the markdown checker workflow back to the branch.  That's all we need, I think.